models/ants.py

```diff
@@ -0,0 +1,9 @@
+from peewee import *
+from .colony import Colony
+from .base_model import BaseModel
+
+class Ants(BaseModel):
+    colony_id = ForeignKeyField(Colony, to_field="id")
+    name = CharField()
+    life_stage = CharField()
```

Comment:
This can be translated into an enum in the program itself, and kept as an integer in the database.